### PR TITLE
Update ca.m3u

### PR DESCRIPTION
**Pull request by CoolCanuck**
Would it be beneficial to separate out the local channels? eg (ca_local.m3u) like there is for USA?

### CODE REVIEW

File: streams/ca.m3u [hunk 1]
```diff
@@ -65,9 +65,9 @@ https://live.streams.ovh/NetvToronto/NetvToronto/playlist.m3u8
 https://pe-fa-lp03a.9c9media.com/live/NOOVO/p/hls/00000201/716cf4c845225692/manifest.m3u8
 #EXTINF:-1 tvg-id="NRBTV.ca" status="online",NRB TV (1080p) [Not 24/7]
 https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/nrb-eu.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="OntarioParliamentaryNetwork.ca" status="online",Ontario Parliamentary Network (720p)
+#EXTINF:-1 tvg-id="OntarioParliamentaryNetwork.ca" status="online",Ontario Parliamentary Network (720p) - English
```
Comment (freearhey) [inline streams/ca.m3u:68]:
```suggestion
#EXTINF:-1 tvg-id="OntarioParliamentaryNetwork.ca" status="online",Ontario Parliamentary Network (English) (720p)
```

File: streams/ca.m3u [hunk 2]
```diff
@@ -65,9 +65,9 @@ https://live.streams.ovh/NetvToronto/NetvToronto/playlist.m3u8
...
 https://origin-http-delivery.isilive.ca/live/_definst_/ontla/house-en/playlist.m3u8
-#EXTINF:-1 tvg-id="OntarioParliamentaryNetwork.ca" status="online",Ontario Parliamentary Network
+#EXTINF:-1 tvg-id="OntarioParliamentaryNetwork.ca" status="online",Ontario Parliamentary Network - French
```
Comment (freearhey) [inline streams/ca.m3u:70]:
```suggestion
#EXTINF:-1 tvg-id="OntarioParliamentaryNetwork.ca" status="online",Ontario Parliamentary Network (French)
```

File: streams/ca.m3u [hunk 3]
```diff
@@ -173,3 +173,67 @@ https://cbcrclinear-tor.akamaized.net/hls/live/2042763/CBCRCLINEAR_TOR_9/master5
...
+https://origin-http-delivery.isilive.ca/live/CHCOTV/live/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",CityNews Calgary / Edmonton / Vancouver / Winnipeg
+https://citynewsregional.akamaized.net/hls/live/1024053/Regional_Live_8/master.m3u8
+#EXTINF:-1 tvg-id="" status="online",CityNews Toronto
+https://citynewsregional.akamaized.net/hls/live/1024052/Regional_Live_7/master.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Barrie ON
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m32_card3/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Borden / Alliston ON
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m38_card2/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Rogerstv - Brantford/Cambridge/Kitchener/Waterloo Region ON
```
Comment (freearhey) [inline streams/ca.m3u:186]:
```suggestion
#EXTINF:-1 tvg-id="" status="online",Rogerstv Rogerstv Brantford/Cambridge/Kitchener/Waterloo Region ON
```

File: streams/ca.m3u [hunk 4]
```diff
@@ -173,3 +173,67 @@ https://cbcrclinear-tor.akamaized.net/hls/live/2042763/CBCRCLINEAR_TOR_9/master5
...
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m37_card4/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Dufferin-Caledon ON
+https://5e7c60955a878.streamlock.net/rtv_live/m40_card8/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Durham Region ON
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m25_card3/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Gander NL
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m37_card1/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Georgina ON
+https://5e7c60955a878.streamlock.net/rtv_live/m39_card3/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Grand Falls - Windsor NL
```
Comment (freearhey) [inline streams/ca.m3u:200]:
```suggestion
#EXTINF:-1 tvg-id="" status="online",Rogerstv Grand Falls Windsor NL
```

File: streams/ca.m3u [hunk 5]
```diff
@@ -173,3 +173,67 @@ https://cbcrclinear-tor.akamaized.net/hls/live/2042763/CBCRCLINEAR_TOR_9/master5
...
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m19_card2/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Kincarde ON
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m19_card1/chunklist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv London ON
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m20_card1/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Miramichi NB
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m43_card3/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv Moncton NB
+https://livedrm.akamaized.net/hls/live/1021686/rtv/m43_card7/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rogerstv NorthDurham - Kawartha ON
```
Comment (freearhey) [inline streams/ca.m3u:214]:
```suggestion
#EXTINF:-1 tvg-id="" status="online",Rogerstv NorthDurham Kawartha ON
```

File: streams/ca.m3u [hunk 6]
```diff
@@ -11,7 +11,7 @@ https://live.livestreamtv.ca/azstar/ngrp:azstar/playlist.m3u8
 http://start.agmediachandigarh.com/gaundapunjab/tv/playlist.m3u8
 #EXTINF:-1 tvg-id="BelmediaTV.ca" status="online",Belmedia TV (1080p)
 https://ythls.onrender.com/channel/UCtj2xA6lW9BPE4BxnhxQQkw.m3u8
-#EXTINF:-1 tvg-id="BNNBloomberg.ca" status="online",BNN Bloomberg
+#EXTINF:-1 tvg-id="BNNBloomberg.ca" status="online",BNN Bloomberg - Event
```
Comment (freearhey) [inline streams/ca.m3u:14]:
```suggestion
#EXTINF:-1 tvg-id="BNNBloomberg.ca" status="online",BNN Bloomberg (Event)
```

**Review by freearhey — CHANGES_REQUESTED** [review]
Dashes in the title are not allowed, as it may cause a bug in some players:

image

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme

**Comment by CoolCanuck** [comment]
Thank you @freearhey . Sorry for not reading that first. I think I accepted the changes properly. Please let me know if I did not do it properly
It is interesting that "[Geo-blocked]" is permitted but not "-" itself. I will be sure to do things properly next time. Thank you for fixing it

**Comment by freearhey** [comment]
As I understand the bug occurs only in cases where the app takes the title as the name of the music track (`<artist name> - <song title>`), so if there are no spaces before and after the dash all works fine.

**Review by freearhey — APPROVED** [review]

**Comment by CoolCanuck** [comment]
closing as I will redo the pull request once the refactoring/sorting is done of all playlists :)

**Pull request closed without merging**